resources/keywords/create_provider_account.resource

```diff
@@ -36,55 +41,130 @@ Application Navigate To Create Provider Account Page And Error Message Displayed
+    Set Suite Variable    ${isv}
+    User Enters Provider Account Name And Selects Database Provider
     Wait Until Page Contains Element    ${txtBxMongoDBOrgID}    timeout=10s
     Input Text    ${txtBxMongoDBOrgID}    ${MONGO.ORG_ID}
     Input Text    ${txtBxMongoDBPubAPI}    ${MONGO.PUB_KEY}
     Input Text    ${txtBxMongoDBPrivAPI}    ${MONGO.PRI_KEY}
     Element Should Be Visible    ${btnCreateEnabled}
     Click Element    ${btnCreateEnabled}
 
-Provider Account Creation Successful And Application Navigates To Success Screen
+User Enters Invalid Data To Create MongoDB Provider Account
+    [Arguments]    ${isv}=MongoDB Atlas Cloud Database Service
+    Set Suite Variable    ${isv}
+    User Enters Provider Account Name And Selects Database Provider    True
+    Wait Until Page Contains Element    ${txtBxMongoDBOrgID}    timeout=10s
+    Input Text    ${txtBxMongoDBOrgID}    ${INV_ORG_ID}
+    Input Text    ${txtBxMongoDBPubAPI}    ${INV_PUB_KEY}
+    Input Text    ${txtBxMongoDBPrivAPI}    ${INV_PRI_KEY}
+    Element Should Be Visible    ${btnCreateEnabled}
+    Click Element    ${btnCreateEnabled}
+
+User Enters Data To Create CrunchyDB Provider Account
+    [Arguments]    ${isv}=Crunchy Bridge managed PostgreSQL
+    Set Suite Variable    ${isv}
+    User Enters Provider Account Name And Selects Database Provider
+    Wait Until Page Contains Element    ${txtBxCrunchyPubAPI}    timeout=10s
+    Input Text    ${txtBxCrunchyPubAPI}    ${CRUNCHY.PUB_KEY}
+    Input Text    ${txtBxCrunchyPrivAPI}    ${CRUNCHY.PRI_KEY}
+    Element Should Be Visible    ${btnCreateEnabled}
+    Click Element    ${btnCreateEnabled}
+
+User Enters Invalid Data To Create CrunchyDB Provider Account
+    [Arguments]    ${isv}=Crunchy Bridge managed PostgreSQL
+    Set Suite Variable    ${isv}
+    User Enters Provider Account Name And Selects Database Provider    True
+    Wait Until Page Contains Element    ${txtBxCrunchyPubAPI}    timeout=10s
+    Input Text    ${txtBxCrunchyPubAPI}    ${INV_PUB_KEY}
+    Input Text    ${txtBxCrunchyPrivAPI}    ${INV_PRI_KEY}
+    Element Should Be Visible    ${btnCreateEnabled}
+    Click Element    ${btnCreateEnabled}
+
+Provider Account Creation Success
     Application Navigates To Provider Account Creation Success Screen
-    Provider Account Creation Success
+    Provider Account Creation Success Page
+    Provider Account Available on Database Access Screen
 
 Application Navigates To Provider Account Creation Success Screen
     Wait Until Keyword Succeeds     5x     2s      Wait Until Page Contains Element    ${lblDBInstSuccess}
     Element Should Be Visible    ${btnViewProviderAcc}
 
-Provider Account Creation Success
+Provider Account Creation Success Page
     Click Element    ${btnViewProviderAcc}
-    Wait Until Page Contains Element    ${btnLastUpdatedHeader}     timeout=10s
+    Wait Until Page Contains Element    ${titleDbaasInvertories}    timeout=10s
```

Comment:
I think there is a spelling error here:  ${titleDbaasInvertories}. I think it should be  ${titleDbaasInventories}?

Reply:
Thanks @ikanias, I have changed it to ${titleDbaasInventories} everywhere.
Pushed with latest commit